map/src/test/java/openlr/map/teststubs/LineStub.java

```diff
@@ -0,0 +1,238 @@
+package openlr.map.teststubs;
+
+import com.vividsolutions.jts.geom.GeometryFactory;
```

Comment:
you still have vividsolutions.jts here

Reply:
After reviewing the dependency tree I think vividsolutions comes from other dependency and we need to keep it. Forget this comment then